_posts/2018-02-05-14-03-caseInsensitiveAndQualifiedFunctions.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title : "14.3 WebApi URI parser default setting updates: case-insensitive name token and unqualified functions & actions"
+description: "7.x WebApi Uri parser using case-insensitve name token and unqualified functions & actions"
+category: "14. 7.x Features "
+---
+
+
+**OData Core Library** v7.x has introduced the following two usability improvement:
+
+* Uri parsing with case-insensitive name token, and 
+
+* Unqualified functions & actions, which are not required to have namespace prefix.
+
+Starting v7.0, these two functionality are supported by default in WebApi.
+ 
+### Examples
+**Prior to WebApi v7.0**, for example, the following Uris are functional:
+
+* GET /service/Customers?$filter=Id eq 5
+
+* POST /service/Customers(5)/Default.UpdateAddress()
+
+With WebApi **v7.0** **by default**, in addition to above, the following variances are also supported:
+
+* GET /service/Customers?$filter=id eq 5
+
+* GET /service/CUSTOMERS?$filter=Id eq 5
+
+* POST /service/Customers(5)/UpdateAddress()
+
+and the combination of both case-insensitive and unqualified functions & actions, such as:
+ 
+* POST /service/CUSTOMERS(5)/UpdateAddress()
+
+
+### Backward Compatibility
+Case-insensitive semantics is supported for type name, property name and function/action name. It will first try to resolve the name token with case-sensitive semantics and return the best match if found; otherwise case-insensitive semantics would be attempted, returning the unique match or throwing exception in case of multiple case-insensitive matches.
```

Comment:
>throwing exception in case of  [](start = 292, length = 30)

replace "throwing exception in case of multiple case-insensitive matches." with "throwing an exception if multiple case-insensitive matches exist."